src/polytopes.jl

```diff
@@ -193,7 +193,7 @@ function bridge(p::Polygon{Dim,T}; width=zero(T)) where {Dim,T}
   end
 
   # retrieve chains as vectors of coordinates
-  pchains = [coordinates.(vertices(c)) for c in chains(p)]
+  pchains = [coordinates.(vertices(c).data) for c in chains(p)]
```

Comment:
We can't access internal fields like this.

Reply:
I know it's risky. The alternative is copying everything into a vector with `collect` (which I did before this commit), which is slow but more stable. Unfortunately I don't know of any way to access the wrapped array present in a `CircularArray`. I can revert to `collect`ing the vertices instead if you prefer.

Reply:
Another alternative is to just use version 1.1 of CircularArrays and not worry about it. But I fear it's not stable on the long-term either. Or we can change to not using CircularArrays and handle wrapping indices ourselves.

Reply:
`coordinates.(::CircularArray{<:Point})` returns a `CircularArray` which doesn't work with the rest of the code.

Reply:
Got it, because the length of the CircularArrays is off by 1? With the first point repeated? I will try to take a look, the `collect` may be the most reasonable solution in this case indeed.

Reply:
I am not sure exactly why. Maybe something like this. If you want to test the code with the `push!` functionality added to CircularArrays feel free to use [my fork](https://github.com/serenity4/CircularArrays.jl).